recipes/ccfits/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ccfits" %}
+{% set version = "2.5" %}
+{% set sha256 = "e542f39fa496417c401be7fbca715d566462cf4c56c83e3d43b6df3cb8d92105" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: CCfits-{{version}}.tar.gz
+  url: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/CCfits-{{version}}.tar.gz
```

Comment:
Is this available on pypi or github? (It's fine if it isn't, it's just that those might be easier to maintain (especially if auto version bumping gets off the ground))

Reply:
No, it is not, unfortunately...